### PR TITLE
Publish updates from master

### PR DESCRIPTION
**Pull request by usha-mandya**
Publish the latest changes from master

### CODE REVIEW

File: docker-for-windows/troubleshoot.md [hunk 1]
```diff
@@ -81,6 +81,27 @@ Diagnostics Bundle: C:\Users\User\AppData\Local\Temp\CD6CF862-9CBD-4007-9C2F-5FB
...
+### Experimental self-diagnose tool
+
+Docker Desktop 3.6.0 contains an experimental "self-diagnose" tool which helps identify some common
+problems. To run it: first locate `com.docker.diagnose.exe`, usually installed in `C:\Program
+Files\Docker\Docker\resources\com.docker.diagnose.exe`.
+
+To run the self-diagnose tool in Powershell:
+
+```powershell
+  PS C:\> & "C:\Program Files\Docker\Docker\resources\com.docker.diagnose.exe" check
```
Comment (thaJeztah) [inline docker-for-windows/troubleshoot.md:93]: Looks like there's some indenting here that we should remove (not urgent)

```suggestion
PS C:\> & "C:\Program Files\Docker\Docker\resources\com.docker.diagnose.exe" check
```

**Review by thaJeztah — APPROVED** [review]
LGTM, but perhaps we can also include https://github.com/docker/docker.github.io/pull/13329 (small change)

File: docker-for-windows/troubleshoot.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-windows/troubleshoot.md:93]: Thanks. Will fix in a follow-up PR

**Review by thaJeztah — APPROVED** [review]
still LGTM

File: docker-for-windows/troubleshoot.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-windows/troubleshoot.md:93]: Fixed in https://github.com/docker/docker.github.io/pull/13332

**Pull request merged**